### PR TITLE
webui: Font awesome

### PR DESCRIPTION
**Pull request by benwa**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

This PR moves most (there might be a few stragglers with third-party libraries) Glyphicons to Font Awesome.  For future proofing, Bootstrap 4 will not include Glyphicons and the Font Awesome project has more options that can be used.

### CODE REVIEW

File: html/includes/print-alert-rules.php [hunk 1]
```diff
@@ -205,9 +205,9 @@
 
     echo '<i>'.htmlentities($rule['rule']).'</i></td>';
     echo '<td>'.$rule['severity'].'</td>';
-    echo "<td><span id='alert-rule-".$rule['id']."' class='glyphicon glyphicon-".$ico.' glyphicon-large text-'.$col."'></span> ";
+    echo "<td><span id='alert-rule-".$rule['id']."' class='fa fa-fw fa-3x fa-".$ico.' text-'.$col."'></span> ";
```
Comment (laf) [inline html/includes/print-alert-rules.php]: 3x seems a bit big :)

File: html/includes/print-alert-templates.php [hunk 2]
```diff
@@ -79,9 +79,11 @@
...
-        echo "<button type='button' class='btn btn-warning btn-sm' data-toggle='modal' data-target='#attach-alert-template' data-template_id='".$template['id']."' name='attach-alert-template'><span class='glyphicon glyphicon-th-list' aria-hidden='true'></span></button>";
+        echo "<div class='btn-group btn-group-sm' role='group'>";
+        echo "<button type='button' class='btn btn-primary btn-sm' data-toggle='modal' data-target='#alert-template' data-template_id='".$template['id']."' data-template_action='edit' name='edit-alert-template'><i class='fa fa-lg fa-pencil' aria-hidden='true'></i></button> ";
+        echo "<button type='button' class='btn btn-danger btn-sm' data-toggle='modal' data-target='#confirm-delete-alert-template' data-template_id='".$template['id']."' name='delete-alert-template'><i class='fa fa-lg fa-trash' aria-hidden='true'></i></button> ";
+        echo "<button type='button' class='btn btn-warning btn-sm' data-toggle='modal' data-target='#attach-alert-template' data-template_id='".$template['id']."' name='attach-alert-template'><i class='fa fa-th-list' aria-hidden='true'></span></button>";
```
Comment (laf) [inline html/includes/print-alert-templates.php]: For some reason I see two fa-th-list's, one in yellow and one floating to the right of it.

File: html/includes/table/alertlog.inc.php [hunk 3]
```diff
@@ -52,33 +52,33 @@
...
     }//end if
     $response[] = array(
         'id'          => $rulei++,
         'time_logged' => $alertlog['humandate'],
-        'details'     => '<a class="glyphicon glyphicon-plus incident-toggle" style="display:none" data-toggle="collapse" data-target="#incident'.($rulei).'" data-parent="#alerts"></a>',
+        'details'     => '<a class="fa fa-plus incident-toggle" style="display:none" data-toggle="collapse" data-target="#incident'.($rulei).'" data-parent="#alerts"></a>',
         'hostname'    => '<div class="incident">'.generate_device_link($dev, shorthost($dev['hostname'])).'<div id="incident'.($rulei).'" class="collapse">'.$fault_detail.'</div></div>',
         'alert'       => htmlspecialchars($alertlog['alert']),
-        'status'      => "<b><span class='glyphicon glyphicon-".$glyph_icon."' style='color:".$glyph_color."'></span> $text</b>",
+        'status'      => "<b><i class='fa fa-".$glyph_icon." text-".$glyph_color."'></i> $text</b>",
```
Comment (laf) [inline html/includes/table/alertlog.inc.php]: Should use $fa_icon here and $fa_color.

File: html/includes/print-alert-rules.php (hunk 1 above)
Comment (benwa) [inline html/includes/print-alert-rules.php]: image
Top is original, middle is `fa-3x`, bottom is `fa-2x`

Comment (benwa) [inline html/includes/print-alert-rules.php]: Works for me

**Comment by laf** [comment]
image

**Pull request merged**